docs/setup.rst

```diff
@@ -39,34 +39,39 @@ or just download the tarball and go that route:
 Installation & Configuration
 ----------------------------
 
-You can go multiple routes with setting up and running Paperless. The `Vagrant
-route`_ is quick & easy, but means you're running a VM which comes with memory
-consumption etc. We also `support Docker`_, which you can use natively under
-Linux and in a VM with `Docker Machine`_ (this guide was written for native
-Docker usage under Linux, you might have to adapt it for Docker Machine.)
-Not to forget the virtualenv, this is similar to `bare metal`_ with the
-exception that you have to activate the virtualenv first.
-Last but not least, the standard `bare metal`_ approach is a little more
-complicated, but worth it because it makes it easier should you want to
-contribute some code back.
-
-.. _Vagrant route: setup-installation-vagrant_
-.. _support Docker: setup-installation-docker_
-.. _bare metal: setup-installation-standard_
+You can go multiple routes with setting up and running Paperless:
+
+ * The `bare metal route`_
+ * The `vagrant route`_
+ * The `docker route`_
+
+
+The `vagrant route`_ is quick & easy, but means you're running a VM which comes
+with memory consumption, cpu overhead etc. The `docker route`_ offer the same
+simplicity a vagrant with less ressources consumption. Only the native usage
```

Comment:
* "a vagrant" should be "as Vagrant"
* "less ressources" should be "lower resource"
* I don't understand what you're trying to say with `Only the native usage under linux is documented you will need to adapt the documentation when using Docker Machine`.